pkg/asset/asset.go

```diff
@@ -168,14 +168,14 @@ func UnwrapCAIResource(asset map[string]interface{}) (*unstructured.Unstructured
 // ConvertToAdmissionRequest converts a CAI asset containing a K8S type to an AdmissionRequest which is the format that
 // the Gatekeeper Constraint Framework target expects.
 func ConvertToAdmissionRequest(asset map[string]interface{}) (*admissionv1beta1.AdmissionRequest, error) {
-	resource, err := UnwrapCAIResource(asset)
+	resource, err := ConvertCAIToK8s(asset)
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to unwrap k8s resource from CAI asset")
+		return nil, errors.Wrapf(err, "failed to convert CAI asset to k8s resource")
 	}
 
 	resourceJSON, err := json.Marshal(resource.Object)
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to convert unwrapped resource to JSON")
+		return nil, errors.Wrapf(err, "failed to convert k8s resource to JSON")
```

Comment:
may i suggest a bit more descriptive error msg: "failed to marshal CAI asset's converted k8s resource to JSON"

Reply:
done in 6780a70